src/main/kotlin/com/jetbrains/snakecharm/lang/psi/SMKParamsReference.kt

```diff
@@ -29,13 +24,10 @@ class SMKParamsReference(
     }
 
     override fun getVariants(): Array<Any> {
-        val variants = mutableListOf<LookupElement>()
-        variants.addAll(getKeywordArguments()
+        return (getKeywordArguments()
                 ?.map { arg ->
                     LookupElementBuilder.create(arg.keyword!!).withIcon(PlatformIcons.PARAMETER_ICON)
-                } ?: emptyList())
-        return variants.toTypedArray()
-
+                } ?: emptyList()).toMutableSmartList().toTypedArray()
```

Comment:
Why do we need to copy result to some intermediate `toMutableSmartList()` list ?